integration_tests/dbt_project.yml

```diff
@@ -29,10 +29,8 @@ vars:
     jira_user_identifier: "user"
     jira_version_identifier: "version"
 
-    jira_schema: jira_source_integration_tests
-
 seeds:
-  jira_source_integration_tests:
```

Comment:
Why was this renamed? I am pretty sure this should match the same name as line 1 in this file. The name of the project as opposed to the name of the schema variable.

Reply:
You can actually see in the BuildKite job that the renamed of the seeds config is causing the job to not recognize the seed files and is not picking them up.
![image](https://github.com/fivetran/dbt_jira_source/assets/74217849/e4a4ad06-b8d8-4d24-bd08-3745ed6e0de7)

Reply:
That was a mistake on my part, forgot to run locally and validate. Reverted.